compiler-plugin/src/main/kotlin/com/google/devtools/ksp/KotlinSymbolProcessingPlugin.kt

```diff
@@ -202,6 +200,14 @@ enum class KspCliOption(
     "log dirty files",
     false,
     false
+    ),
+
+    CHANGED_CLASSES_OPTION(
+    "changedClasses",
+    "<changedClasses>",
+    "dirty classes in classpath",
```

Comment:
nit: Add if it is internal or canonical name (using / or . as separators).

Reply:
Done. ce0b3b1bceef5fcf7f2b4b4f47364336f91b8713